README.md

```diff
@@ -0,0 +1,21 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello World
+
+This project is my first mini project as a micronaut.
+
+This project is just an index HTML page that displays the text 'Hello Microverse.
+
+
+
+
+## Built With
+
+- HTML
+- CSS
+
```

Comment:
- [ ] Please add more sections in `Readme`  file, for example you can add Authors and Acknowledgements sections 

```
###Authors
NAME

GitHub: [github.com/d2ndjim](https://github.com/YOURGITHUB)
Twitter: [@d2ndjim_](https://twitter.com/YOURTWITTER)
LinkedIn: [linkedin.com/in/lekanj](https://linkedin.com/in/YOURLINKEDIN)

### Acknowledgments
@Microverseinc for creating this project.
```